src/app/achievements/achievements.component.html

```diff
@@ -9,6 +9,24 @@
     <ion-spinner></ion-spinner>
   </ng-container>
   <ion-grid>
+    <ion-row class="total-points">
```

Comment:
Need some `*ngIf` as the wrapper, in case there's no achievements in this program